src/vs/base/browser/ui/findinput/findInput.ts

```diff
@@ -74,6 +75,7 @@ export class FindInput extends Widget {
 	protected regex: RegexToggle;
 	protected wholeWords: WholeWordsToggle;
 	protected caseSensitive: CaseSensitiveToggle;
+	protected additionalToggles: Toggle[] = [];
```

Comment:
👏 

We had a different implementation for additional in Notebook Find Widget (passing in an action which can open a submenu https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts#L159 ) 

<img width="317" alt="image" src="https://user-images.githubusercontent.com/876920/179432815-1e868d8b-7061-4d80-860f-46a30788417b.png">

We might need some efforts in debt week to unify this before we have more variations (https://github.com/microsoft/vscode/issues/144851)

Reply:
Interesting, I couldn't find that. I was surprised to find so many different find inputs, find widgets, etc.